src/WindowManager/WindowManager.ts

```diff
@@ -66,9 +68,12 @@ export class WindowManager {
       throw new ErrCannotEstablishHandshake();
     }
 
+    this._session = this._session || isWalletReady.data || '';
```

Comment:
rename isWalletReady to something non-boolean-ish

Reply:
maybe `handshakeResponse` ?